### DISCUSSION TITLE
Huion H640P

### DISCUSSION THREAD

**Discussion by Braxtyboy03** [OpenTabletDriver/OpenTabletDriver#2623] [Q&A] [answered]
Does anyone know how to fix this problem? So the pen and tablet are connected on OpenTabletDriver and it works perfectly fine until i open a game other than osu! it doesn't let me shoot or anything not even use the mouse on the home screen i would really like to play it on rainbow six siege so can anyone please tell me a solution if they know of one?

**Comment by InfinityGhost** [comment, marked as answer]
You may need to use the [Vmulti](https://opentabletdriver.net/Wiki/FAQ/Windows#vmulti) plugin, some anticheats are known to block the SendInput API that we use by default.

**Reply by jamesbt365** [reply]
https://opentabletdriver.net/Wiki/FAQ/Windows#vmulti
